utils/db.ts

```diff
@@ -296,6 +302,45 @@ export function newCommentProps(): Pick<Comment, "id" | "createdAt"> {
 }
 
 export async function createComment(comment: Comment) {
+  const itemKey = ["items", comment.itemId];
+  const itemsByUserKey = ["items_by_user", comment.userId, comment.itemId];
+  const [itemRes, itemsByUserRes] = await kv.getMany<Item[]>([
+    itemKey,
+    itemsByUserKey,
+  ]);
+
+  if (itemRes.value === null || itemsByUserRes.value === null) {
+    throw new Error(`Item not found`);
+  }
+
+  const itemsByTimeKey = [
+    "items_by_time",
+    itemRes.value.createdAt.getTime(),
+    itemRes.value.id,
+  ];
+  const itemsByTimeRes = await kv.get<Item>(itemsByTimeKey);
+
+  if (itemsByTimeRes.value === null) throw new Error(`Item not found`);
+
+  const newItemRes = itemRes.value;
+  const newItemsByTimeRes = itemsByTimeRes.value;
+  const newItemsByUserRes = itemsByUserRes.value;
+
+  newItemRes.commentsCount++;
+  newItemsByTimeRes.commentsCount++;
+  newItemsByUserRes.commentsCount++;
+
+  const com = await kv.atomic()
+    .check(itemRes)
+    .check(itemsByTimeRes)
+    .check(itemsByUserRes)
+    .set(itemKey, newItemRes)
+    .set(itemsByTimeKey, newItemsByTimeRes)
+    .set(itemsByUserKey, newItemsByUserRes)
```

Comment:
Can you please ensure all operations in this function happen on the same atomic operation (below)?